src/App.tsx

```diff
@@ -12,18 +12,13 @@ export const App: React.FC<{}> = () => (
   <StylesProvider injectFirst>
     <MuiThemeProvider theme={theme}>
       <ThemeProvider theme={theme}>
-        <Router>
-          <Switch>
-            <Route exact path="/">
-              <HomePage />
-            </Route>
-            {ITProfessions.map(profession => (
```

Comment:
We have no need for a separate `Route` per each profession - we're actually using the same component for all professions (`DetailPage`) so a single `Route` is just fine! :) Also, given that our professions have unique routes, we can use path params to get the information about the current profession.